file-formats/sito/examples/zmo_sito_test.sito.json

```diff
@@ -0,0 +1,262 @@
+          }
+        ],
+        "structureValueHelpMap": [
+          {
+            "appType": "allApps",
+            "type": "service",
+            "scope": "sap",
+            "fieldName": "SITNDEMOCARRIERID"
+          },
+          {
+            "appType": "allApps",
+            "type": "service",
+            "scope": "sap",
+            "fieldName": "SITNDEMOPLANETYPE"
+          },
+          {
+            "appType": "allApps",
+            "type": "service",
+            "scope": "sap",
+            "fieldName": "SITNDEMOCARRIERREGION"
+          },
+          {
+            "appType": "allApps",
+            "type": "service",
+            "scope": "sap",
+            "fieldName": "SITNDEMOCONNECTIONID"
+          }
+        ],
+        "structureValueHelpMapSrv": [
+          {
+            "type": "allApps",
+            "fieldName": "SITNDEMOCARRIERID",
+            "entitySet": "C_Sitn2DemoFlightVH",
+            "property": "SitnDemoCarrierID",
+            "id": "SITDEMO_VH"
+          },
+          {
+            "type": "allApps",
+            "fieldName": "SITNDEMOPLANETYPE",
+            "entitySet": "C_Sitn2DemoFlightVH",
+            "property": "SitnDemoPlaneType",
+            "id": "SITDEMO_VH"
+          },
+          {
+            "type": "allApps",
+            "fieldName": "SITNDEMOCARRIERREGION",
+            "entitySet": "C_Sitn2DemoFlightVH",
+            "property": "SitnDemoRegionCode",
+            "id": "SITDEMO_VH"
+          },
+          {
+            "type": "allApps",
+            "fieldName": "SITNDEMOCONNECTIONID",
+            "entitySet": "C_Sitn2DemoFlightVH",
+            "property": "SitnDemoConnectionID",
+            "id": "SITDEMO_VH"
+          }
+        ]
+      }
+    ],
+    "situationObjectAction": [
+      {
+        "type": "callbackAction",
+        "scope": "sap",
+        "id": "SITDEMO_ACT_CANCEL_FLIGHT",
+        "actionText": [
+          {
+            "name": "Cancel Flight",
+            "description": "Set flight status to unscheduled"
+          }
+        ],
+        "actionCallback": [
+          {
+            "callbackId": "SITDEMO_CB_CANCEL_FLIGHT"
+          }
+        ]
+      },
+      {
+        "type": "callbackAction",
+        "scope": "sap",
+        "id": "SITDEMO_ACT_CHANGE_PLANE",
+        "actionText": [
+          {
+            "name": "Change Plane",
+            "description": "Assign another plane to the flight"
+          }
+        ],
+        "actionCallback": [
+          {
+            "callbackId": "SITDEMO_CB_CHANGE_PLANE"
+          }
+        ]
+      },
+      {
+        "type": "navigationAction",
+        "scope": "sap",
+        "id": "SITDEMO_ACT_MANAGE_AIRLINE",
+        "actionText": [
+          {
+            "name": "Display Airline",
+            "description": "Show contact details"
+          }
+        ],
+        "actionNavigation": [
+          {
+            "id": "SITDEMO_NAV_MANAGE_AIRLINE"
```

Comment:
Just a remark: Here you use just `id` (which I think I prefer). for the `actionCallback`, you used `callbackId`.

Reply:
This was missed last time. Fixed now.